packages/@lwc/ssr-compiler/src/compile-template/transformers/component.ts

```diff
@@ -79,7 +79,7 @@ const bYieldFromChildGenerator = esTemplateWithYield`
 `<EsBlockStatement>;
 
 const bAddContent = esTemplate`
```

Comment:
```suggestion
// Note that this function name (`generateSlottedContent`) does not need to be scoped even though
// it may be repeated multiple times in the same scope, because it's a function _expression_ rather
// than a function _declaration_, so it isn't available to be referenced anywhere.
const bAddContent = esTemplate`
```